lib/lrama/option_parser.rb

```diff
@@ -59,6 +59,7 @@ def parse_by_option_parser(argv)
         o.on('-S', '--skeleton=FILE', 'specify the skeleton to use') {|v| @options.skeleton = v }
         o.on('-t', 'reserved, do nothing') { }
         o.on('--debug', 'display debugging outputs of internal parser') {|v| @options.debug = true }
+        o.on('--no-stdlib', 'do not include standard library') {|v| @options.no_stdlib = true }
```

Comment:
I prefer grammar file declaration like `%no-stdlib` to command line options because grammar file declaration is more declarative than command line option. It's difficult to track what options are passed from parse.y file.
What do you think about it?

Reply:
It would certainly be better that way. I agree with your opinion. I will change it.